rlclientlib/logger/http_transport_client.h

```diff
@@ -41,6 +43,8 @@ namespace reinforcement_learning {
     ~http_transport_client();
   protected:
     int v_send(const buffer& data, api_status* status) override;
+    int v_send(const buffer& data, unsigned int event_counts, api_status* status) override;
```

Comment:
It seems like abstraction leakage here. Maybe it is possible to introduce original_event_count as a field of event batch fralbuffers message and track it there?